js/apps/system/_admin/aardvark/APP/frontend/js/views/documentsView.js

```diff
@@ -1090,7 +1090,12 @@
       var self = event.currentTarget;
 
       var url;
-      var doc = $(self).attr('id').substr(4);
+      var doc;
+      if ($(self).attr('id')) {
+        doc = $(self).attr('id').substr(4);
+      } else {
+        return;
+      }
```

Comment:
The following would be easier to read:
```suggestion
      if (!$(self).attr('id')) {
        return;
      }
      var doc = $(self).attr('id').substr(4);
```